Bezusov_3LessonHW.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+    <script>
+    // 1)
+            //  var a = 'block';
+            //     if (a === 'block') { 
+            //             console.log('block')
+            //         } else if (a === 'none') {
+            //             console.log('none')
+            //         } else if (a === 'inline') {
+            //             console.log('inline')
+            //         } else {
+            //             console.log('other')
+            // }    
+    // ReWriteHere to the Switch
+            // var a = 'none';
+            // switch(a) {
+            //     case 'block':
+            //             console.log('block');
+            //         break;
+            //     case 'none':
+            //         console.log('none');
+            //         break;
+            //     case 'inline':
+            //         console.log('inline');
+            //         break;
+            //     default: console.log('other');
+            // }
+    // 2) переписать в тернарные 
+        // 1)
+                // var block = 'hidden';
+                // block = 'hidden' 
```

Comment:
it's wrong. Code doesn't work in correct way. `block = block === 'hidden' ? 'visible' : 'hidden'`